### PR TITLE
Fix plugin test failures

### PR DESCRIPTION
**Pull request by gimantha**
## Purpose

Relates to https://github.com/ballerina-platform/module-ballerina-graphql/pull/2107

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility
- [ ] No `commons` package changes (if there are any, please update the GraphQL version in [GraphQL tools](https://github.com/ballerina-platform/graphql-tools) and [Ballerina dev tools](https://github.com/ballerina-platform/ballerina-dev-tools))
- [ ] No `compiler` package changes (if there are any, please update the GraphQL version in [Ballerina dev tools](https://github.com/ballerina-platform/ballerina-dev-tools))

### CODE REVIEW

File: compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java [hunk 1]
```diff
@@ -985,7 +985,7 @@ public void testAnonymousRecordsAsFieldType() {
 
         diagnostic = diagnosticIterator.next();
         message = getErrorMessage(CompilationDiagnostic.INVALID_ANONYMOUS_FIELD_TYPE,
-                                  "record {|string name; int age;|}", "Query.class.profile");
+                                  "record {|string name; int age;|}", "Query.'class.profile");
```
Comment (ThisaruGuruge) [inline compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java:988]: This quote is invalid. Is there a place we missed to escape?

```suggestion
                                  "record {|string name; int age;|}", "Query.class.profile");
```

File: compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java [hunk 2]
```diff
@@ -1044,7 +1044,7 @@ public void testInvalidUseOfReservedFederatedTypeNames() {
         Iterator<Diagnostic> diagnosticIterator = diagnosticResult.errors().iterator();
 
         Diagnostic diagnostic = diagnosticIterator.next();
-        String message = getErrorMessage(CompilationDiagnostic.INVALID_USE_OF_RESERVED_TYPE_AS_OUTPUT_TYPE, "Query.any",
+        String message = getErrorMessage(CompilationDiagnostic.INVALID_USE_OF_RESERVED_TYPE_AS_OUTPUT_TYPE, "Query.'any",
```
Comment (ThisaruGuruge) [inline compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java:1047]: Ditto

File: compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java (hunk 1 above)
Comment (gimantha) [reply compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java:988]: @ThisaruGuruge, this is compile time error, as the resource path has the quote, it is now visible in errors also. Runtime behavior is not changed

Comment (gimantha) [reply compiler-plugin-tests/src/test/java/io/ballerina/stdlib/graphql/compiler/ServiceValidationTest.java:988]: Had a discussion with @DimuthuMadushan .. will reevaluate

**Pull request closed without merging**